modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/GETTING_STARTED.markdown

```diff
@@ -0,0 +1,186 @@
+# Getting Started with Liferay Workspace
+
+Complete documentation for Liferay Workspace can be found 
+[here](https://learn.liferay.com/dxp/7.x/en/developing-applications/tooling/liferay-workspace.html).
+
+### Folder Structure
+```
+my-project
+├── configs
+│   ├── common
+│   ├── dev
+│   ├── docker
+│   ├── local
+│   ├── prod
+│   └── uat
+├── ext
+│   ├── core ext
+│   └── module exts
+├── modules
+│   ├── apis
+│   ├── services
+│   ├── java widgets
+│   └── js widgets
+├── themes
+│   └── js themes
+└── wars
```

Comment:
from my conversations with upgrade customers they wanted the wars along side the modules, so a service-builder portlet gets split up into 3 api/service/portlet but we would move the "portlet" into the "wars" folder, far away from the other 2 conceptually linked projects.  Thus I feel its best if we just move wars under 'modules'

Reply:
is `liferay.workspace.wars.dir` still valid?

Reply:
sounds good lets make the adjustment to the newest best practices